main.py

```diff
@@ -103,6 +103,15 @@ async def on_chat(msg):
                         await send_message(f"@{user.get_username()}: {to_do}", bot_chat)
                     else:
                         console.log("Cancelled due to cooldown")
+                
+                elif msg.text.lower().startswith("!repeat"):
+                    msg_text = msg.text[19:]
+                    console.log(f"Repeating {user.get_username()}")
+                    if tell_me_to_cooldown.run():
+                        await send_message(f"{msg_text}", bot_chat)
```

Comment:
I think the user that used the command should be mentioned somewhere, especially for Ryver organizations that allow users to remove their own messages.

Reply:
1: I definitely agree with that, we wouldn't want anyone using profanity and then making it look like BrainBot just did it out of nowhere. I'll try and add something for that.

2: Quick question, what does msg.text[19:] mean?

Reply:
It takes the message text and gets everything after the 19th character.
```py
>>> text = "abcde"
>>> text[3:]
'de'
>>> text[:3]
'abc'
```